client/src/pages/courses.tsx

```diff
@@ -33,18 +47,30 @@ export default function CoursesPage() {
             </Helmet>
             <Grid container spacing={2}>
                 <Grid item xs={isSmallScreen ? 12 : 7}>
-                    <VideoComponent
-                        playlistTitle={courses[0].title}
-                        videoTitle={courses[0].title2}
-                        videoUrl={courses[0].videoUrl}
-                        description={courses[0].description}
-                        videoImage={courses[0].image}
-                    />
+                    {selectedClass?.file ? (
+                        <VideoComponent
+                            playlistTitle={courses[0].title}
+                            videoTitle={selectedClass?.title ?? 'Loading...'}
+                            videoUrl={selectedClass?.file ?? ''}
+                            description={
+                                selectedClass?.description ?? 'Loading...'
+                            }
+                            videoImage={
+                                'https://d33v4339jhl8k0.cloudfront.net/docs/assets/591c8a010428634b4a33375c/images/5ab4866b2c7d3a56d8873f4c/file-MrylO8jADD.png'
```

Comment:
If the thumbnail is always going to be the same, since I believe that the image attribute of a class does not exist, I think it is better to download said image and save it in the "client/public/src/static" folder or something similar, and reference it here

Reply:
I have added the image in .svg format as Javier advised, for the responsiveness of the page. The problem is that when using vectors the image is shown in black and white, let me know what you think and if I don't use the .png, file that I have also saved in that folder in case it is necessary.